feat(changelog): transactional-email-changed-tem-in-ga 2023-05-10 (#1633)
scaleway/docs-content: changelog/may2023/2023-05-10-transactional-email-changed-tem-in-ga.mdx

---

> title: TEM in GA

Reviewer: ```suggestion
title: Transactional Email is in General Availability!
```

status: changed
author:
    fullname: 'Join the #transactional-email channel on Slack.'

>     url: 'https://scaleway-community.slack.com/archives/C0358JE0XEK'

Reviewer 2: ```suggestion
    url: 'https://slack.scaleway.com'
```

date: 2023-05-10
category: managed-services
product: transactional-email
---

> Scaleway TEM (Transactional email) is now in General Availability! TEM is a fully managed solution dedicated to sending your transactional emails.

Reviewer: ```suggestion
Scaleway Transactional Email (TEM) is now in General Availability! TEM is a fully managed solution dedicated to sending your transactional emails. Check out our [dedicated documentation](https://www.scaleway.com/en/docs/managed-services/transactional-email/).
```

> Scaleway Transactional Email (TEM) is now in General Availability! TEM is a fully managed solution dedicated to sending your transactional emails. Check out our [dedicated documentation](https://www.scaleway.com/en/docs/managed-services/transactional-email/).

Reviewer 3: ```suggestion
Scaleway Transactional Email is now in General Availability! Transactional Email is a fully managed solution dedicated to sending your transactional emails. Check out our [dedicated documentation for more information](/managed-services/transactional-email/).
```
